db/providers/notesProvider.js

```diff
@@ -0,0 +1,39 @@
+import dotenv from "dotenv";
+
+import Note from "../../models/notes";
+import { LIMIT_NUMBER } from "../../setters/getters/calculations/uppercase constant";
+import { postfix } from "../../utils/milieu";
+
+dotenv.config();
+
+export const getNote = async () => {
+  const note = await Note.find({ title: "tiger" }, {})
```

Comment:
Let's name filtering by dynamic title. We will need to send it in query string params and controller will get it and in case it's exist, filtering notes by title.